### PR TITLE
Test math operators (ceil, floor, exp, log, reciprocal, sigmoid, sqrt, tanh)

### PR DESCRIPTION
**Pull request by tianleiwu**

### CODE REVIEW

File: docs/Operators.md [hunk 1]
```diff
@@ -3785,6 +3890,26 @@ opset_import {
...
+<details>
+<summary>reciprocal</summary>
+
+```python
+node = onnx.helper.make_node(
+    'Reciprocal',
+    inputs=['x'],
+    outputs=['y'],
+)
+x = np.random.randn(3, 4, 5).astype(np.float32)
```
Comment (bddppq) [inline docs/Operators.md]: Probably `+ 2` to go away from 0?

Comment (tianleiwu) [reply docs/Operators.md]: randn is normal distribution so there is still chance x < -2. 
I added a line to avoid element with value 0:
    x[x==0] = 1.

Comment (bddppq) [reply docs/Operators.md]: Elements close to zero will have issues too. Use `np.random.rand(...) + 1`?

**Comment by houseroad** [comment]
One simple suggestion, because the case test also serves as sample cases, it's better to put some explicit tensors (instead of using random generator) for the input and output. Also we can also write the expected results explicitly in the comments.

**Comment by houseroad** [comment]
something like:
```
def test_add:
    a = [1, 2, 3]
    b = [3, 4, 5]
    expected = np.add(a, b) # expected output: [4, 6, 8]
```

**Comment by tianleiwu** [comment]
For document, I agree that explicit example could be helpful for understanding.  

Explicit tensors can be achieved by another way: during generating test data, the tool also dumps input and output tensors as example, instead of putting test code as example.

In this way, we can decouple test code and example.

**Comment by tianleiwu** [comment]
Added manually crafted input tensors for test examples.

**Pull request merged**